variables.tf

```diff
@@ -172,19 +172,19 @@ variable "schemas" {
     enable_console_output                         = optional(bool, false)
     pipe_execution_paused                         = optional(bool, false)
     stages = optional(map(object({
-      enabled              = optional(bool, true)
-      descriptor_name      = optional(string, "snowflake-stage")
-      aws_external_id      = optional(string)
-      comment              = optional(string)
-      copy_options         = optional(string)
-      credentials          = optional(string)
-      directory            = optional(string)
-      encryption           = optional(string)
-      file_format          = optional(string)
-      snowflake_iam_user   = optional(string)
-      storage_integration  = optional(string)
-      url                  = optional(string)
-      create_default_roles = optional(bool)
+      enabled                    = optional(bool, true)
+      descriptor_name            = optional(string, "snowflake-stage")
+      aws_external_id            = optional(string)
+      comment                    = optional(string)
+      copy_options               = optional(string)
+      credentials                = optional(string)
+      directory                  = optional(string)
+      encryption                 = optional(string)
+      file_format                = optional(string)
+      snowflake_iam_user         = optional(string)
+      storage_integration        = optional(string)
+      url                        = optional(string)
+      create_default_stage_roles = optional(bool)
```

Comment:
We're keeping that unified everywhere, so rollback to `create_default_roles` instead of `create_default_stage_roles`